samples/nrf9160/modem_shell/README.rst

```diff
@@ -344,6 +344,44 @@ Examples
 
 ----
 
+Locaton tool
+============
+
+MoSh command: ``location``
+
+You can use the Location tool for retrieving devices location with different location methods.
+See :ref:`lib_location` library for information on the configuration of different location methods and services
+but there are some available by default to ease trials.
```

Comment:
```suggestion
See :ref:`lib_location` library for information on the configuration of different location methods and services.
Default configurations are available to facilitate trials.
```

Reply:
Pekka made little modification to your suggestion so I accepted that.